src/Twig/TagsExtention.php

```diff
@@ -13,20 +13,20 @@
 
 namespace MonsieurBiz\SyliusAdminBetterLoginPlugin\Twig;
 
-use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\DependencyInjection\ParameterBag\ParameterBagInterface;
 use Twig\Extension\AbstractExtension;
 use Twig\TwigFunction;
 
 class TagsExtention extends AbstractExtension
 {
     /**
-     * @var ContainerInterface
+     * @var ParameterBagInterface
      */
-    private $container;
+    private $parameterBag;
 
-    public function __construct(ContainerInterface $container)
+    public function __construct(ParameterBagInterface $parameterBag)
```

Comment:
You should either bump the required Symfony version or allow both `ContainerInterface` and `PrameterBagInterface` here and add a BC layer in the constructor to handle both.

Reply:
Hi @dunglas! I think your suggestion isn't necessary in our case. The ParameterBagInterface is available since Symfony 4.1, and the Sylius 1.8 version uses Symfony 4.4. What do you think about it?

Reply:
I suggest upgrading to a major version with an upgrade file: https://github.com/monsieurbiz/SyliusAdminBetterLoginPlugin/pull/9/commits/0588c273cab858dc1558c797e90b0248013c0f5e